awx_collection/plugins/modules/tower_user.py

```diff
@@ -45,7 +45,7 @@
       type: str
     is_superuser:
       description:
-        - User is a system wide administrator.
```

Comment:
I'd like to add the ability to have a vars/descriptions.yml file which can house descriptions that do not currently come from Tower. Maybe we have some kind of warning if the vars/descriptions.yml overrides a non-'' value from Tower.

Reply:
hopefully those could be shared between multiple modules. But yeah, that's possible.